applications/values.yaml

```diff
@@ -67,7 +67,11 @@ applications:
     imageTag: *gitApiVersion
     config:
       hookFile:
-        path: /schema
+        path: /runtime
+        name: webhooks.yaml
+      configFile:
+        path: /runtime
+        name: lodestar-runtime-config.yaml
```

Comment:
It doesn't seem like the values in the `config` hash will change much between environments. I would suggest to move the `config` hash to the `values.yaml` file in the `lodestar-git-api` repo. This way they become defaults and can be overridden here  if required plus less configuration that lives in this file.

Reply:
Moved `config` values to the git-api repo and updated description of this PR.